src/blockrelay/graphene_set.cpp

```diff
@@ -29,31 +30,41 @@ CGrapheneSet::CGrapheneSet(size_t _nReceiverUniverseItems,
 
     FastRandomContext insecure_rand(fDeterministic);
 
+    // Infer various receiver quantities
+    uint64_t nReceiverExcessItems =
+        std::max((int)(nReceiverUniverseItems - nItems), (int)(nSenderUniverseItems - nItems));
+    nReceiverExcessItems = std::max(0, (int)nReceiverExcessItems); // must be non-negative
+    nReceiverExcessItems =
+        std::min((int)nReceiverUniverseItems, (int)nReceiverExcessItems); // must not exceed total mempool size
+    uint64_t nReceiverMissingItems = std::max(1, (int)(nItems - (nReceiverUniverseItems - nReceiverExcessItems)));
```

Comment:
why is 1 the maximum expected missing transactions?

Reply:
I just kept that as a buffer (this assumption goes back to the original implementation as you can see [here](https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/7e0f47a7aa6f72f0c573a6ea1aa298b43a2c38e1/src/blockrelay/graphene_set.cpp#L112)). It makes the IBLT a little more robust in case the receiver is actually missing some transactions that we didn't account for. There is nothing fundamental about adding this padding though. I could run some tests where it is set to 0 to see if it greatly affects decode rate.